### PR TITLE
toxic with audio working

### PR DESCRIPTION
**Pull request by its-pointless**
I tested this and got voice at the other end...

### CODE REVIEW

File: packages/openal-soft/build.sh [hunk 1]
```diff
@@ -1,10 +1,34 @@
...
+TERMUX_PKG_SRCURL=https://github.com/kcat/openal-soft/archive/$_commit.zip
+#TERMUX_PKG_SRCURL=http://kcat.strangesoft.net/openal-releases/openal-soft-${TERMUX_PKG_VERSION}.tar.bz2
+TERMUX_CMAKE_BUILD=Ninja
 TERMUX_PKG_EXTRA_CONFIGURE_ARGS="
--DALSOFT_UTILS=OFF
--DALSOFT_EXAMPLES=OFF
+-DALSOFT_UTILS=ON
+-DALSOFT_EXAMPLES=ON
 -DALSOFT_TESTS=OFF
+-DHAVE_PULSEAUDIO=OFF
```
Comment (tomty89) [inline packages/openal-soft/build.sh]: Let's enable pulse support as well?
```
diff --git a/Alc/backends/pulseaudio.c b/Alc/backends/pulseaudio.c
index 8c5469b2..3d9ca039 100644
--- a/Alc/backends/pulseaudio.c
+++ b/Alc/backends/pulseaudio.c
@@ -188,6 +188,8 @@ static ALCboolean pulse_load(void)
 #define PALIB "libpulse-0.dll"
 #elif defined(__APPLE__) && defined(__MACH__)
 #define PALIB "libpulse.0.dylib"
+#elif defined(__ANDROID__)
+#define PALIB "libpulse.so"
 #else
 #define PALIB "libpulse.so.0"
 #endif
```

**Comment by its-pointless** [comment]
No. Mainly so we don't force people to download pulseaudio unless necessary.

**Comment by tomty89** [comment]
You only need libpulseaudio as build dependency. It loads the library dynamically.

**Comment by fornwall** [comment]
Nice! Can we use the new 1.19.0 version of openal-soft (http://openal-soft.org/#release) now instead of building from a commit?

**Comment by its-pointless** [comment]
yes
Only issue is phone mic isn't working within toxic. Still trying to figure out openal issue properly.

**Comment by its-pointless** [comment]
this is probably good to go?

**Comment by Neo-Oli** [comment]
@its-pointless Why did you close this?

**Comment by its-pointless** [comment]
because its out of date and the openal-soft patch doesn't work. I will submit a working version. I should also mention the new patch upstream to maintainer.

**Pull request closed without merging**